packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -226,6 +229,11 @@ export interface IContainerRuntimeOptions {
 
     // Override summary configurations
     summaryConfigOverrides?: Partial<ISummaryConfiguration>;
+
+    // Flag that disables putting channels in isolated subtrees for each data store
+    // and the root node when generating a summary if set to true.
+    // Defaults to TRUE (disabled) for now.
+    disableIsolatedChannels?: boolean;
```

Comment:
i'd prefer this defaulted to false, so `enableIsolatedChannels?: true`, then you don't need to enable the property to disable the feature in this.runtimeOptions below. i also dislike truthy checks in general, so with my suggestion all checks become `enableIsolatedChannels === true` as false and undefined are treated the same way

Reply:
Agree- but the plan is to change the default behavior from true (disabled) to false (enabled) after a few releases. I've encapsulated the checks into the getter, so it should only be doing the weird truthy logic in one place.